source/common/loadCollector.js

```diff
@@ -20,6 +20,10 @@ var cpuCollector = function (period, onLoad) {
         var cpus = os.cpus();
         var idle = 0;
         var total = 0;
+        if (!cpus || !olds) {
```

Comment:
If olds is undefined, error would be thrown at line 18,  "var end = olds.length - 1".
So in your case, the os.cpus() cannot be obtained always, or just in line 20?

Reply:
Yes，I have encountered this situation cpus is undefined several times, but never encounter olds is undifined.
In my case，owt modules crash may need to run for a long time.

Reply:
I mean, change the condition to "if (!cpus)" if olds is never undefined.